2d_PSF_code/Zernike_Module.py

```diff
@@ -179,9 +179,11 @@
 from scipy.ndimage.filters import gaussian_filter
 import scipy.fftpack
 import scipy.misc
+from scipy.interpolate import interp2d
 from scipy.special import erf
 from astropy.convolution import Gaussian2DKernel
 from astropy.convolution import Tophat2DKernel
+from scipy.interpolate import interp1d
```

Comment:
I suggest putting this scipy.interpolate import just below the previous scipy.interpolate import